tests/unit/test_disable_compilation.py

```diff
@@ -0,0 +1,140 @@
+import pytest
+from spyre_util import write_sample_model_config
+from vllm.config import (ModelConfig, ParallelConfig, SchedulerConfig,
+                         VllmConfig)
+
+from vllm_spyre.platform import (PRE_COMPILE_MODEL_CATALOG_FILENAME,
+                                 SpyrePlatform)
+
+
+@pytest.mark.parametrize("batch_type", ["sb", "cb"])
```

Comment:
We usually have sb/cb parameterization like this in other tests:
```
parametrize(
            "cb", [pytest.param(1, marks=pytest.mark.cb, id="cb"), 0])
```
where cb ==1 means cb else sb.

Would it help if we follow the same convention? If so, the `tests/conftest.py` file automatically adds this parameterization if we just add an argument called `cb` within the test function name and remove the existing parameterization.

Reply:
surely. I'll try that out. I took this convention from the base PR of this change #455